### ISSUE TITLE
NPC's are not visible

### ISSUE THREAD

**Issue by OldTownMC** [CitizensDev/Citizens2#380]
Using Spigot 1.8.3 NPC's are created, but are not visible. I tested this on two diffrent dedicated servers (linked together using bungeecord thou), and it doesn't work. I then tested it on my other server which isn't related to that bungeecord, and on a seperate dedicated server and it did work. I then proceded to remove all plugins except for Citizens from the server that it doesn't work on, and it still did not work. 

Both server spigot versions are the same.

**Comment by mcmonkey4eva** [comment]
NPCs are, i nfact, visible, if you're running latest dev build of Citizens on a 1.8.3 server.

Have no idea what you're doing, but you're doing it wrong.

**Comment by OldTownMC** [comment]
Yes, as I stated. On one of my servers it is visible, but on the second they are not at all. I'll try and look into it some more.

**Comment by OldTownMC** [comment]
Okay some more info: I can assign commands to it, select it, but its not visible. I cannot equip it though... 

Okay, figured some stuff out. It works when you change the type, but it doesn't work with the type "player".

**Comment by mcmonkey4eva** [comment]
Did you get it working?

**Comment by OldTownMC** [comment]
No, still doesn't work with players, yet works with all mobs.

**Comment by Morphan1** [comment]
@OldTownMC could you pastebin your startup log for us?

**Comment by mcmonkey4eva** [comment]
Will reopen when you're able to provide more information.

**Comment by OldTownMC** [comment]
So I disconnected one of my servers connected to the bungeecord with a human NPC, and it started working. So for some reason I believe its related to this: For some odd reason, quote someone else: The servers seems to send PlayerListEntry packets before sending the JoinGame packet. And I guess it may be related, as the thing I stated above creates issues with other things (idk even know why it happens).
